src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -157,17 +157,24 @@ LedgerEntryHandler::process(LedgerEntryHandler::Input input, Context const& ctx)
         return Error{*status};
 
     auto const lgrInfo = std::get<ripple::LedgerHeader>(lgrInfoOrStatus);
-    auto const ledgerObject = sharedPtrBackend_->fetchLedgerObject(key, lgrInfo.seq, ctx.yield);
-
-    if (!ledgerObject || ledgerObject->empty())
-        return Error{Status{"entryNotFound"}};
+    auto output = LedgerEntryHandler::Output{};
+    output.deletedLedgerIndex = std::nullopt;
+    auto ledgerObject = sharedPtrBackend_->fetchLedgerObject(key, lgrInfo.seq, ctx.yield);
```

Comment:
const?

Reply:
`ledgerObject` is modified again below. But maybe there is no need to do that and it would be ok to just create a new variable in the loop, maybe hide it like `if (auto const ledgerObject = ...; !ledgerObject.isEmpty() or ...)`.